src/fu-util-common.c

```diff
@@ -2864,6 +2864,17 @@ fu_util_print_builder(FuConsole *console, JsonBuilder *builder, GError **error)
 	return TRUE;
 }
 
+void
+fu_util_print_message_as_json(FuConsole *console, const gchar *message)
```

Comment:
I don't think I like *message* -- I think the json either has to be an error (e.g. with a domain, code, and message) or a boolean success value.

Reply:
Maybe in the cases that it's just a string we shouldn't have anything? Just use the application return code of

Reply:
I think so too.